fmriprep/cli/run.py

```diff
@@ -345,11 +375,15 @@ def main():
             errno = 1
         else:
             raise
-
-    # Generate reports phase
-    errno += generate_reports(subject_list, output_dir, work_dir, run_uuid)
-    write_derivative_description(bids_dir, str(Path(output_dir) / 'fmriprep'))
-    sys.exit(int(errno > 0))
+    finally:
+        # Generate reports phase
+        errno += generate_reports(subject_list, output_dir, work_dir, run_uuid,
+                                  sentry_sdk=sentry_sdk)
+        write_derivative_description(bids_dir, str(Path(output_dir) / 'fmriprep'))
+
+        if not opts.notrack and errno == 0:
```

Comment:
(just to prepare the next suggestion)

```suggestion
        if not opts.notrack:
```